eng/Versions.props

```diff
@@ -129,7 +129,9 @@
     <SystemDrawingCommonVersion>8.0.0</SystemDrawingCommonVersion>
     <SystemIOFileSystemAccessControlVersion>5.0.0</SystemIOFileSystemAccessControlVersion>
     <SystemMemoryVersion>4.5.5</SystemMemoryVersion>
-    <SystemReflectionMetadataVersion>7.0.0</SystemReflectionMetadataVersion>
+    <SystemReflectionMetadataVersion>9.0.0-alpha.1.24072.1</SystemReflectionMetadataVersion>
+    <!-- Keep toolset versions in sync with dotnet/msbuild and dotnet/sdk -->
```

Comment:
These aren't the versions in msbuild or VS afaik

Reply:
https://github.com/dotnet/msbuild/blob/main/eng/Versions.props#L38-L39

Reply:
Yes, see https://github.com/dotnet/runtime/pull/97021#issuecomment-1904515180 and https://github.com/dotnet/runtime/pull/97643#discussion_r1473069272. TL;DR, we need to keep HostModel version lower or equal (when everything is out of beta) than MSBuild and SDK. SDK is currently using 7.0, MSBuild has moved to 8.0.